RasimKhusaenov/my_own_blog
# Redesign comments

### Summary

- Comments sector redesigned
- Feature test was refactored, because there is no label now

### How it works

![image](https://user-images.githubusercontent.com/35406008/175364931-347527e9-8034-47e5-88a7-8e69b6dcb2bb.png)
![image](https://user-images.githubusercontent.com/35406008/175365832-0816db9e-1156-4dae-a767-510defaf9f02.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in
- Go to https://my-own-blog-pr-64.herokuapp.com/articles/2
- Make sure that comments displayed correctly and there is no scroll when your comment text is too big.

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #63 Issue
- GitHub Pull Request #62
